launch_lightup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+BRANCH=${LIGHTUP_BRANCH:-main}
+DOWNLOAD_LOCATION=${LIGHTUP_DOWNLOAD_LOCATION:-"https://s3.us-west-2.amazonaws.com/www.lightup.ai/releases/lupmgr"}
```

Comment:
Easy change for the future: I suggest we create another bucket purely for lupmgr releases.

Reply:
+1